styles/_user-profiles.scss

```diff
@@ -99,6 +99,11 @@
   outline-color: $brand_color;
 }
```

Comment:
This rule can be deleted now

Reply:
It's not really part of my changes, though?

Reply:
oh, you mean the entire CSS class

Reply:
But we still support `{{edit}}` for the time being, so shouldn't we keep it around?

Reply:
yes, class sorry, as the `.edit-profile-button` will no longer be used

Reply:
> But we still support {{edit}} for the time being, so shouldn't we keep it around?

If it is not used in the CPH theme we don't need to have it here. Custom themes keep their own CSS.

Reply:
Ah, makes sense 👍

Reply:
Pushed the removal of it